ui-frontend/packages/catalog-ui-search/src/main/webapp/component/query-title/query-title.view.js

```diff
@@ -84,7 +84,6 @@ module.exports = Marionette.LayoutView.extend({
   },
   handleTitleUpdate() {
     this.$el.find('input').val(this.model.get('title'))
-    this.updateQueryName()
   },
   updateQueryName(e) {
     this.$el.find('.button-title').html(this.getSearchTitle() + zeroWidthSpace)
```

Comment:
can we add this line to the `handleTitleUpdate`? 

```
 this.$el.find('.button-title').html(this.getSearchTitle() + zeroWidthSpace)
```
We missed this in the 2.19.x but there's [another pr](https://github.com/codice/ddf/pull/6463) open to add it in.